website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+
+    ```sql 
+    select * from jaffle_shop.customers;
+    select * from jaffle_shop.orders;
+    select * from stripe.payment;
+    ```
+## Connect dbt Cloud to Redshift 
+1. Create a new project in [dbt Cloud](https://cloud.getdbt.com/). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
+2. Enter a project name and click **Continue**.
+3. For the warehouse, click **Redshift** then **Next** to set up your connection.
+4. Enter your Redshift settings. Reference your credentials you saved from the CloudFormation template.
+    - **Hostname** &mdash; Your entire hostname. Make sure to drop the `http://` at the beginning and any trailing slashes at the end.
+    - **Port** &mdash; `5439`
+    - **Database** &mdash; `dbtworkshop`.
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/redshift_tutorial/images/dbt_cloud_redshift_account_settings.png" title="dbt Cloud - Redshift Cluster Settings" />
+    </div>
+
+5. Set your development credentials. These credentials will be used by dbt Cloud to connect to Redshift. Those credentials (as provided in your CloudFormation output) will be:
+    - **Username** &mdash; `dbtadmin`
+    - **Password** &mdash; This is the password that you set earlier in the guide
+    - **Schema** &mdash; dbt Cloud automatically generates a schema name for you. By convention, this is `dbt_<first-initial><last-name>`. This is the schema connected directly to your development environment, and it's where your models will be built when running dbt within the Cloud IDE.
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/redshift_tutorial/images/dbt_cloud_redshift_development_credentials.png" title="dbt Cloud - Redshift Development Credentials" />
+    </div>
+
+6. Click **Test Connection**. This verifies that dbt Cloud can access your Redshift cluster.
+7. Click **Next** if the test succeeded. If it failed, you might need to check your Redshift settings and credentials.
+
+## Set up a dbt Cloud managed repository 
+<Snippet src="tutorial-managed-repo" />
+
+## Initialize your dbt project​ and start developing
+Now that you have a repository configured, you can initialize your project and start development in dbt Cloud:
+
+1. Click **Develop** from the upper left. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
+2. Above the file tree to the left, click **Initialize dbt project**. This builds out your folder structure with example models.
+3. Make your initial commit by clicking **Commit & Sync**. Use the commit message `initial commit` and click **Commit**. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
+4. You can now directly query data from your warehouse and execute `dbt run`. You can try this out now:
+    - In the IDE's editor, add this query: 
```

Comment:
Instructions not clear on where in the editor do we add this query. At this point in the instructions we have a barebones setup. Executing `dbt run` works without this as well. 
```suggestion
    - In the IDE's editor, add this query: 
```

Reply:
great catch! updated to add "click **Create new file** " for people to add that query into